dashboard/modules/job/utils.py

```diff
@@ -237,3 +238,43 @@ async def find_job_by_ids(
         return job
 
     return None
+
```

Comment:
nit: Should the previous function above just call this new one with a single ID? To dedup code.

Reply:
I purposefully wanted to reimplement so that it batched the call to the GCS. This API will be called frequently every 5 seconds for the UI.